lib/internal/Magento/Framework/Image/Adapter/ImageMagick.php

```diff
@@ -77,7 +82,11 @@ public function open($filename)
         try {
             $this->_imageHandler = new \Imagick($this->_fileName);
         } catch (\ImagickException $e) {
-            throw new \Exception(sprintf('Unsupported image format. File: %s', $this->_fileName), $e->getCode(), $e);
+            throw new LocalizedException(
+                new \Magento\Framework\Phrase(sprintf('Unsupported image format. File: %s', $this->_fileName)),
```

Comment:
Whole line can be replaced with `__('Unsupported image format. File: %1', $this->_fileName),`